005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -1 +1,162 @@
+    \end{tabular}
+\end{table}
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{@{}lllll@{}}
+    \toprule
+    HP inc last tick [nonfriend]    & Stubbornness      & Morality         & Mood             & Friendship     \\ \midrule
+    ask Food Taken                  & 0                 & 0                & 0                & -1             \\
+    ask HP                          & -1                & 1                & 1.5              & 1              \\
+    ask Intended Food Intake        & 0                 & 0                & 0                & 0              \\
+    Request Take Food               & 0                 & 0                & 0                & 0              \\
+    Request Leave Food              & -1                & 1                & 1.5              & 1              \\ \bottomrule
+    \end{tabular}
+\end{table}
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{@{}lllll@{}}
+    \toprule
+    Sum total                       & Stubbornness      & Morality         & Mood             & Friendship     \\ \midrule
+    Request Leave Food              & 0                 & 0                & 0                & 0              \\ \bottomrule
+    \end{tabular}
+\end{table}
+This zero-sum agent is how our agent is currently set, it represents the neutral agent. One who possesses no inherent emotional bias. This was done by balancing the positive and negative reactions to a friend, and those to a non-friend separately. We noticed the need for this when our agent would initially get stuck at the +ve and -ve limits of certain emotions. As future work, it would be exciting to see the emergent strategies which we hope to observe as we introduce biases. For instance, an agent whose net sum of all interactions leaves them with a stubbornness increase, coupled with an inherently moral agent, may create strategies yet thought of. 
+
+\subsection{Proposing a Treaty}
+When the agent HP is critical, they will ask for help by either asking for the person upstairs to leave at least the amount of food necessary to survive or, if it doesn’t have any active treaties, a treaty that states: if HP is higher than 20 leave 95\% of the food in the platform for 3 days. At this time the agent's self-preservation instinct takes precedence and their objective becomes to eat enough food to get out of the critical state before the agent dies.
+When the agent is not in critical condition, it will be able to propose other treaties depending on their morality. The first treaty is detrimental for the tower and the other two are ‘fair’ treaties for the tower. \par
+\begin{enumerate}
+    \item If the agent is very immoral, it will send a treaty that is detrimental for anyone above them. It is expected for this treaty to be rejected by any sensible agent’s and is done as a way of mimicking frustration or the desire to hurt others when morality is low.
```

Comment:
No comma on agent's

Reply:
```suggestion
    \item If the agent is very immoral, it will send a treaty that is detrimental for anyone above them. It is expected for this treaty to be rejected by any sensible agents and is done as a way of mimicking frustration or the desire to hurt others when morality is low.
```